libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -60,7 +60,7 @@ pair<string, string> IRGenerator::run(
 	{
 		string errorMessage;
 		for (auto const& error: asmStack.errors())
-			errorMessage += langutil::SourceReferenceFormatter::formatErrorInformation(*error);
+			errorMessage += langutil::SourceReferenceFormatterHuman::formatErrorInformation(*error);
```

Comment:
Why not `withErrorIds`?

Reply:
Not sure, is that useful? If it is we can add it.

Reply:
I didn't mean to change the current behavior, just wondering if this is the same as the current behavior